README.md

```diff
@@ -30,7 +30,7 @@
 
 ## About EddieBotUI
 
-EddieBotUI is a web application built to display all kinds of information about the **Eddie Jaoude Discord community** - [click to join](https://discord.gg/fPss8W9) - like users biography, usage stats, badges and more.
+EddieBotUI is a web application built to display all kinds of information about the **Eddie Jaoude Discord community** - [click to join](https://discord.gg/jZQs6Wu) - like users biography, usage stats, badges and more.
```

Comment:
I think we could use the same link as the other repos, so that we don't need to change in the future 😃. How about this:
```suggestion
EddieBotUI is a web application built to display all kinds of information about the **EddieHub Discord community** - [click to join](http://discord.eddiehub.org) - like users biography, usage stats, badges and more.
```

Reply:
Yeah sure we can merge my PR as I was the first one to solve the issue

Reply:
have you pushed a commit with this fix?

Reply:
> have you pushed a commit with this fix?

yeah I did it in #52